src/components/checkbox/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { ChangeEvent, FC } from 'react';
+import { Checkbox as ChakraCheckbox } from '@chakra-ui/react';
+
+interface Props {
+  name: string;
+  value?: string;
+  isDisabled?: boolean;
+  isChecked?: boolean;
+  onChange?: (event: ChangeEvent<HTMLInputElement>) => void;
+  label?: string;
+  iconColor?: string;
+  isInvalid?: boolean;
+}
+
+const Checkbox: FC<Props> = ({
+  name,
+  value,
+  isDisabled = false,
+  isChecked = false,
+  onChange,
+  label,
+  iconColor = 'gray.900',
```

Comment:
This is a way I found which I could change the color on the icon for the disabled state.
I tried on the container and control style objects but we cannot change the icon color there because it cannot apply the disabled state on those objects.

Reply:
Do I understand correctly that the `_disabled` modifier on the icon does not work as expected? That's strange - it normally should work on the multipart components (at least from what I understand). If we need to resort to custom values, let's make sure this prop is not exposed publicly and just use a variable.

Reply:
I checked again and it doesn't work on the icon (the modifiers don't apply there) but on the control works. Thank you for the push! 🌸 I already removed the UGLY iconColor prop 🧼